raiden/tests/unit/test_api.py

```diff
@@ -0,0 +1,21 @@
+import pytest
```

Comment:
Probably better `test_cli.py` ? `test_cli_interface.py` is somohow redudant naming as cli means command-line interface